index.html

```diff
@@ -196,5 +192,6 @@ <h2>{{ event.title }}</h2>
 			</div>
 			<div class="home-spatial__cta"><a href="/spatial-technologies/">Spatial Tech Home</a></div>
 		</section> <!-- end sections -->
-	</main> <!-- end main content -->
+
+	</div> <!-- end main content -->
```

Comment:
As with opening `main`, need to check switching this from a div doesn't break styling